### PR TITLE
UrlUniformityRule added

### PR DESCRIPTION
**Pull request by gwyndol1ne**

### CODE REVIEW

File: src/main/kotlin/com/github/darderion/mundaneassignmentpolice/rules/Rules.kt [hunk 1]
```diff
@@ -248,3 +249,19 @@ val RULE_SHORTENED_URLS = URLRuleBuilder()
...
+val RULE_URLS_UNIFORMITY=URLRuleBuilder()
+	.called("Ссылки разных видов")
+	.disallow { urls ->
+		val filteredUrls : List<Pair<String, Line>> = urls.filter { pair ->
+			val url = pair.first
+			!url.startsWith("htt")
+		}
+		if (urls.size==filteredUrls.size)
+		{
+			filteredUrls.filter {	pair ->
```
Comment (Darderion) [inline src/main/kotlin/com/github/darderion/mundaneassignmentpolice/rules/Rules.kt:261]: Filter method doesn't change filteredUrls

File: src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/RulesTests.kt [hunk 2]
```diff
@@ -48,6 +48,9 @@ class RulesTests: StringSpec({
 	"Rule should detect incorrect symbols in section names" {
 		RULE_SYMBOLS_IN_SECTION_NAMES.process(PDFBox().getPDF(filePathSymbolsInSectionNames)).count() shouldBeExactly 4
 	}
+	"Rule should detect links of different types" {
+		RULE_URLS_UNIFORMITY.process(PDFBox().getPDF(filePathUniformityUrls)).count() shouldBeExactly 1
```
Comment (Darderion) [inline src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/RulesTests.kt:52]: Test doesn't match rule's violations

**Review by Darderion — CHANGES_REQUESTED** [review]

File: src/main/kotlin/com/github/darderion/mundaneassignmentpolice/rules/Rules.kt (hunk 1 above)
Comment (gwyndol1ne) [reply src/main/kotlin/com/github/darderion/mundaneassignmentpolice/rules/Rules.kt]: updated

File: src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/RulesTests.kt (hunk 2 above)
Comment (gwyndol1ne) [reply src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/RulesTests.kt]: updated

**Pull request merged**